tests/integration/test_hyper_transformer.py

```diff
@@ -1481,6 +1481,44 @@ def test_hypertransformer_with_mutli_column_transformer_end_to_end(self):
         pd.testing.assert_frame_equal(transformed_data, expected_transformed_data)
         pd.testing.assert_frame_equal(reverse_transformed_data, data_test)
 
+    def test_hypertransformer_with_mutli_column_transformer_and_single_column(self):
+        """Test a mutli column transformer used with for a single column."""
+        # Setup
+        data_test = pd.DataFrame({
+            'A': ['1.0', '2.0', '3.0'],
+            'B2': ['4.0', '5.0', '6.0'],
+            'C': [True, False, True]
+        })
+        dict_config = {
+            'sdtypes': {
+                'A': 'categorical',
+                'B2': 'categorical',
+                'C': 'boolean'
+            },
+            'transformers': {
+                'A': DummyMultiColumnTransformerNumerical(),
+                ('B2'): DummyMultiColumnTransformerNumerical(),
```

Comment:
This `('B2')` is `'B2'` , either add `('B2', )` or `set(['B2'])`, unless you are intending to pass just a `string`, then we can remove the `( )`
![Screenshot from 2023-10-31 12-45-08](https://github.com/sdv-dev/RDT/assets/41479552/4c1961f7-409b-4b8d-bb9f-b6f20a587486)